Extension/package.json

```diff
@@ -185,12 +185,6 @@
           "description": "%c_cpp.configuration.formatting.description%",
           "scope": "resource"
         },
-        "C_Cpp.navigation.length": {
-          "type": "number",
-          "default": 60,
-          "description": "%c_cpp.configuration.navigation.length.description%",
```

Comment:
Could you also remove this string from package.nls.json?  Same with any other string (placeholders) in package.json removed and no longer referenced.  (c_cpp.command.navigate.title)